src/code.cloudfoundry.org/gorouter/cmd/gorouter/main.go

```diff
@@ -276,15 +276,15 @@ func main() {
 	os.Exit(0)
 }
 
-// initializeDropsondeReporter setups metrics via dropsonse if enabled
+// initializeDropsondeReporter setups logs, and metrics via dropsonde if enabled
```

Comment:
```suggestion
// initializeDropsondeReporter setups logs and metrics via dropsonde if enabled
```

Reply:
The comma is in there on purpose; `initializeDropsondeReporter` always setups logs, and metrics only if enabled.

Reply:
Maybe make it more explicit. Both fine with me
```suggestion
// initializeDropsondeReporter setups logs. It also sets up metrics via dropsonde if enabled
```